data/json/snippets/radio.json

```diff
@@ -13,17 +14,31 @@
+      "I’m DJ Dustbowl, this is my co-host, Sam, and my survival tip for today, folks, is simply to be resourceful.  This sounds like a whole lot of vague bullshit, I know, but I managed to find another survivor broadcasting on ham radio.  We got to talking, and This crazy mother fucker has apparently made a complete suit of armor from sheet metal padded with ripped-up body pillows, a mace from plumbing pipes, and has retooled their old hatchback into a fucking armored brick.  People, while we may be living in a wasteland, with some creativity, we can still make it ours.  So, I’m going to queue up an appropriate track, Sam, you sign us off, and to all of you listening… chin up and say, \"Welcome to my wasteland!\"",
+      "What’s up, people of the cataclysm?  I’m DJ Dustbowl, this is my partner in crime, Sam, and you’re listening to Dark Days Radio, but… let me shut my mouth for a bit.\n\n\"Hey hey, this is Anarchy, former writer, now turned apocalypse survivor and wearer of oven-derived sheet metal armor.\"\n\nAren’t you forgetting, \"Driver of a hatchback turned death brick?\"\n\n\"Yes.  That’s an important part of my zombie-trashing credentials.\"\n\nSo yeah, ladies and gentlemen, if you didn’t catch my last broadcast, I did the impossible: I found, quite possibly, the only other living, breathing nerd operating a radio on the eastern seaboard.\n\n\"Truly, we are an endangered species.\"\n\nThat we are, but, Anarchy, please enlighten the people bunkering down on how you got here.\n\n\"Let’s just say… a lot of zombies got introduced to the front grill of the death brick.\"\n\nMetal.  Now, folks, I’m going to step out for a smoke, and I’ll leave y’all in Anarchy’s hands.  I’m not responsible for anything that gets said.\n\n\"OK, so I’m going to be bringing you whatever music I feel like playing, as well as tips, tricks, and excerpts from whatever I've been writing in this apocalyptic hellscape.  But before we launch, I’d like to give some dad advice.  Don't forget to work on your personal projects, people.  I know you've got plenty of time sitting in those shelters of yours.\"",
+      "It's time to line up some fresh tracks and for y’all to crank up the volume (don’t do that if you’re in an infested area), because I’m DJ Dust Bowl and you're rocking along to Dark Days Radio.  It’s just me and Sam in the studio again, folks.  Anarchie’s on the road back home, and we just wanna say that it was a pleasure having him join us in manning Dark Days Radio for a while.  So, Anarchy, if you’re driving along and tuning in as well, have a safe trip, you sheet metal-wearing lunatic.  Hallelujah to have had someone like you kick around the set, and hallelujah for all the new tracks you brought in for us!",
```

Comment:
```suggestion
      "It's time to line up some fresh tracks and for y’all to crank up the volume (don’t do that if you’re in an infested area), because I’m DJ Dustbowl and you're rocking along to Dark Days Radio.  It’s just me and Sam in the studio again, folks.  Anarchy’s on the road back home, and we just wanna say that it was a pleasure having him join us in manning Dark Days Radio for a while.  So, Anarchy, if you’re driving along and tuning in as well, have a safe trip in your death brick, you sheet metal-wearing lunatic.  Hallelujah to have had someone like you kick around the set, and hallelujah for all the new tracks you brought in for us!",
```